ompi/mca/pml/ob1/pml_ob1_isend.c

```diff
@@ -13,7 +13,7 @@
  * Copyright (c) 2007-2016 Los Alamos National Security, LLC.  All rights
  *                         reserved.
  * Copyright (c) 2014      Cisco Systems, Inc.  All rights reserved.
- * Copyright (c) 2015      Research Organization for Information Science
```

Comment:
?

Reply:
Not sure, I can undo if desired.

Reply:
I would remove it, I don't see any other change in the file.